pipeline/07-export.R

```diff
@@ -365,22 +415,140 @@ assessment_card_prepped <- assessment_card %>%
 for (town in unique(assessment_pin_prepped$township_code)) {
   message("Now processing: ", town_convert(town))
 
-
-  ## 5.1. PIN-Level ------------------------------------------------------------
+  workbook_name <- glue(
+    params$assessment$year,
+    str_replace(town_convert(town), " ", "_"),
+    "Initial_Model_Values.xlsx",
+    .sep = "_"
+  )
 
   # Filter overall data to specific township
   assessment_pin_filtered <- assessment_pin_prepped %>%
     filter(township_code == town) %>%
     select(-township_code)
 
+  # 5.3 Comp details -----------------------------------------------------------
+
+  # Filter the training data so that we only display sales that are referenced
+  # in comps
+  training_data_filtered_1 <- training_data %>%
+    inner_join(
+      assessment_pin_filtered %>% select(comp_pin_1),
+      by = join_by(meta_pin == comp_pin_1)
+    )
+  training_data_filtered_2 <- training_data %>%
+    inner_join(
+      assessment_pin_filtered %>% select(comp_pin_2),
+      by = join_by(meta_pin == comp_pin_2)
+    )
+  training_data_filtered <- bind_rows(
+    training_data_filtered_1, training_data_filtered_2
+  ) %>%
+    unique()
```

Comment:
**nitpick:** Something like the below might be cleaner:

```suggestion
  training_data_filtered_1 <- training_data %>%
    filter(
      meta_pin %in%
      assessment_pin_filtered %>%
        select(comp_pin_1, comp_pin_2) %>%
        unlist(),
    ) %>%
    distinct()
```

Reply:
Much better, done in e02cdd29b4cc03e629f9f0abeff53a51fccb632b!